.htaccess

```diff
@@ -23,3 +23,15 @@ RedirectMatch 302 \
 RedirectMatch 302 \
 	^/support/security-page/?$ \
 	https://forums.classicpress.net/c/support/security-page
+
+RedirectMatch 302 \
+	^/updating-classicpress/?$ \
+	https://docs.classicpress.net/updating-classicpress/
+
+RedirectMatch 302 \
+	^installing-classicpress/#installation-step \
+	https://docs.classicpress.net/installing-classicpress/#installation-step
+
+RedirectMatch 302 \
+	^testing-classicpress/#reporting-bugs \
```

Comment:
This link and the one above it won't work as written because of the `#hash-fragments`. The browser [does not send any hash fragment to the server with the URL](https://stackoverflow.com/a/3664324/106302), and the practice of linking to random hash fragments is also one of the things we'd like to get rid of with this link.classicpress.net scheme.

The hash fragments should be left in the target URL though, there they will work fine.